timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -3,10 +3,15 @@
 
 import logging
 import os
+import time
+import elasticsearch
```

Comment:
@berggren heads up, not sure if that is okay to use / import here, but below I tried to use the most accurate exception type for es timeout, that is the reason behind importing it here

Reply:
That should be fine